todo/mvnw.cmd

```diff
@@ -0,0 +1,205 @@
+@REM ----------------------------------------------------------------------------
+@REM Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Are all these setup files needed? The idea is to keep this a minimal as possible.

We don't need to worry about deploying it, as long as it can run locally.

Reply:
Yes deleted